parsers/CZ.py

```diff
@@ -68,6 +68,71 @@ def get_target_datetime(dt: datetime | None) -> datetime:
     return dt
 
 
+def fetch_consumption(
```

Comment:
This should not be named fetch_consumption if it don't return the expected consumption element.

Should be renamed to get_consumption